app/models/katello/concerns/operatingsystem_extensions.rb

```diff
@@ -44,6 +44,24 @@ def set_atomic_attributes
       def atomic?
         name.match(/.*atomic.*/i)
       end
+
+      def rhel_eos_schedule
+        return nil unless name == "RedHat" # using name and not title so we get specifically RHEL, not rebuilds
```

Comment:
Is it safe to assume all RHEL OSes use "RedHat" as the name? Or do some use "RHEL" ?

Reply:
Seems they are all `RedHat` for RHEL 9, 8 and 7.

Reply:
@chris1984 do you happen to know what the strings are for RHEL6 and RHEL5? :smile:

Reply:
@jeremylenz Should be good to go with 6 too:

```
irb(main):004:0> Operatingsystem.second
=> #<Redhat id: 6, major: "6", name: "RedHat", minor: "10", nameindicator: nil, created_at: "2023-09-01 13:51:18.161260000 +0000", updated_at: "2023-09-01 13:51:18.161260000 +0000", release_name: nil, type: "Redhat", description: nil, password_hash: [FILTERED], title: "RedHat 6.10">
```

Reply:
_narrator voice_  They were not good to go.